python/ray/serve/backend_worker.py

```diff
@@ -317,6 +345,9 @@ async def invoke_batch(self, request_item_list: List[Query]) -> List[Any]:
                                  "results with length equal to the batch size"
                                  ".".format(batch_size, len(result_list)))
                 raise RayServeException(error_message)
+            for i, result in enumerate(result_list):
+                result_list[i] = await self.ensure_serializable_response(result
```

Comment:
for formatting, you can wrap `(await self.ensure...)` in a paren, i think the formatter will make it prettier.

Reply:
Thanks, it worked!